test/index.js

```diff
@@ -61,7 +61,6 @@ test('getAbi calculates correct Node ABI', function (t) {
 
 test('getAbi calculates correct Electron ABI', function (t) {
   t.throws(function () { getAbi(undefined, 'electron') })
-  t.throws(function () { getAbi('7.2.0', 'electron') })
```

Comment:
Just realized that this changes behavior:
`node-abi` wouldn't throw anymore if the version isn't supported yet. This could lead to strange behaviors since `node-abi` would just return the wrong ABI instead. Let me fix that.